mkdd_widgets.py

```diff
@@ -1227,6 +1233,23 @@ def paintGL(self):
                             point_index += 1
                     glLineWidth(1.0)
 
+                # Mark lap/section checkpoints in a fancy green
```

Comment:
There are some rendering problems at the moment with this approach.

The lines (3D lines with depth test enabled) look a bit awkward:
![image](https://github.com/RenolY2/mkdd-track-editor/assets/1853278/a9564726-b890-4716-80c4-74680c6d904f)

And, when the checkpoint is selected, it becomes thicker (as expected), but turns blue (but it should remain green).

I'll share a patch with you (on Discord), based on your changes, with a potential solution.